external/CMakeLists.txt

```diff
@@ -1,51 +1,96 @@
+    breakpad/src/common/md5.cc 
+    breakpad/src/common/md5.h 
+    breakpad/src/common/string_conversion.cc 
+    breakpad/src/common/string_conversion.h 
+)
+
+set(BREAKPAD_SOURCES_COMMON_LINUX
+    breakpad/src/common/linux/elf_core_dump.cc 
+    breakpad/src/common/linux/elfutils.cc 
+    breakpad/src/common/linux/elfutils.h 
+    breakpad/src/common/linux/file_id.cc 
+    breakpad/src/common/linux/file_id.h 
+    breakpad/src/common/linux/guid_creator.cc 
+    breakpad/src/common/linux/guid_creator.h 
+    breakpad/src/common/linux/linux_libc_support.cc 
+    breakpad/src/common/linux/memory_mapped_file.cc 
+    breakpad/src/common/linux/safe_readlink.cc
+)
+
+set(BREAKPAD_SOURCES_COMMON_LINUX_GETCONTEXT
+    breakpad/src/common/linux/breakpad_getcontext.S
+)
+
+set(BREAKPAD_SOURCES_COMMON_WINDOWS
+    breakpad/src/common/windows/guid_string.cc
+    breakpad/src/common/windows/guid_string.h
+)
+
+set(BREAKPAD_SOURCES_CLIENT_LINUX
+    breakpad/src/client/minidump_file_writer-inl.h 
+    breakpad/src/client/minidump_file_writer.cc 
+    breakpad/src/client/minidump_file_writer.h 
+    breakpad/src/client/linux/crash_generation/crash_generation_client.cc 
+    breakpad/src/client/linux/crash_generation/crash_generation_server.cc 
+    breakpad/src/client/linux/dump_writer_common/thread_info.cc 
+    breakpad/src/client/linux/dump_writer_common/ucontext_reader.cc 
+    breakpad/src/client/linux/handler/exception_handler.cc 
+    breakpad/src/client/linux/handler/exception_handler.h 
+    breakpad/src/client/linux/handler/minidump_descriptor.cc 
+    breakpad/src/client/linux/handler/minidump_descriptor.h 
+    breakpad/src/client/linux/log/log.cc 
+    breakpad/src/client/linux/log/log.h 
+    breakpad/src/client/linux/microdump_writer/microdump_writer.cc 
+    breakpad/src/client/linux/microdump_writer/microdump_writer.h 
+    breakpad/src/client/linux/minidump_writer/linux_core_dumper.cc 
+    breakpad/src/client/linux/minidump_writer/linux_dumper.cc 
+    breakpad/src/client/linux/minidump_writer/linux_ptrace_dumper.cc 
+    breakpad/src/client/linux/minidump_writer/minidump_writer.cc 
+)
+
+set(BREAKPAD_SOURCES_CLIENT_WINDOWS
+    breakpad/src/client/windows/crash_generation/client_info.cc
+    breakpad/src/client/windows/crash_generation/client_info.h
+    breakpad/src/client/windows/crash_generation/crash_generation_client.cc
+    breakpad/src/client/windows/crash_generation/crash_generation_client.h
+    breakpad/src/client/windows/crash_generation/crash_generation_server.cc
+    breakpad/src/client/windows/crash_generation/crash_generation_server.h
+    breakpad/src/client/windows/crash_generation/minidump_generator.cc
+    breakpad/src/client/windows/crash_generation/minidump_generator.h
+    breakpad/src/client/windows/handler/exception_handler.cc
+    breakpad/src/client/windows/handler/exception_handler.h
+    breakpad/src/client/windows/sender/crash_report_sender.cc
+    breakpad/src/client/windows/sender/crash_report_sender.h
```

Comment:
Can we compile without the sender since we have our own?

Reply:
removed unnedded files for Windows from CMakeLists